samples/java_layout/src/main/java/io/customer/android/sample/java_layout/ui/settings/SettingsActivity.java

```diff
@@ -77,30 +74,22 @@ private void parseLinkParams() {
         if (deepLinkUri != null) {
             String cdpApiKey = deepLinkUri.getQueryParameter("cdp_api_key");
             if (cdpApiKey != null) {
-                ViewUtils.setTextWithSelectionIfFocused(binding.cdpApiKeyTextInput, cdpApiKey);
+                ViewUtils.setTextWithSelectionIfFocused(binding.settingsCdpApiKeyLabel, cdpApiKey);
             }
             String siteId = deepLinkUri.getQueryParameter("site_id");
             if (siteId != null) {
-                ViewUtils.setTextWithSelectionIfFocused(binding.siteIdTextInput, siteId);
+                ViewUtils.setTextWithSelectionIfFocused(binding.settingsSiteIdKeyLabel, siteId);
             }
         }
         isLinkParamsPopulated = true;
     }
 
     private void prepareViewsForAutomatedTests() {
         ViewUtils.prepareForAutomatedTests(binding.topAppBar);
-        ViewUtils.prepareForAutomatedTests(binding.deviceTokenTextInput, R.string.acd_device_token_input);
-        ViewUtils.prepareForAutomatedTests(binding.apiHostTextInput, R.string.acd_api_host_input);
-        ViewUtils.prepareForAutomatedTests(binding.cdnHostTextInput, R.string.acd_cdn_host_input);
-        ViewUtils.prepareForAutomatedTests(binding.cdpApiKeyTextInput, R.string.acd_cdp_api_key_input);
-        ViewUtils.prepareForAutomatedTests(binding.siteIdTextInput, R.string.acd_site_id_input);
-        ViewUtils.prepareForAutomatedTests(binding.flushIntervalTextInput, R.string.acd_flush_interval_input);
-        ViewUtils.prepareForAutomatedTests(binding.flushAtTextInput, R.string.acd_flush_at_input);
-        ViewUtils.prepareForAutomatedTests(binding.trackScreensSwitch, R.string.acd_track_screens_switch);
-        ViewUtils.prepareForAutomatedTests(binding.trackDeviceAttributesSwitch, R.string.acd_track_device_attributes_switch);
-        ViewUtils.prepareForAutomatedTests(binding.debugModeSwitch, R.string.acd_debug_mode_switch);
-        ViewUtils.prepareForAutomatedTests(binding.saveButton, R.string.acd_save_settings_button);
-        ViewUtils.prepareForAutomatedTests(binding.restoreDefaultsButton, R.string.acd_restore_default_settings_button);
+        ViewUtils.prepareForAutomatedTests(binding.settingsCdpApiKeyLabel, R.string.acd_cdp_api_key_input);
+        ViewUtils.prepareForAutomatedTests(binding.settingsSiteIdKeyLabel, R.string.acd_site_id_input);
+        ViewUtils.prepareForAutomatedTests(binding.settingsSaveButton, R.string.acd_save_settings_button);
+        ViewUtils.prepareForAutomatedTests(binding.settingsRestoreDefaultsButton, R.string.acd_restore_default_settings_button);
```

Comment:
@Shahroz16 @mrehan27 I couldn't find those UI tests, do I need to update something for the new fields?

Reply:
The tests are available in a separate Appium repository. We haven't maintained them in a while though, but having unique and consistent values across all sample apps should be sufficient for now.